frontend/components/flashcard/Flashcard.tsx

```diff
@@ -49,6 +50,10 @@ interface FlashcardProps {
 	 * callback to push rating
 	 */
 	push?: (body: { cardID: string; rating: number }) => void;
+	/**
+	 * callback to push rating
+	 */
+	role?: keyof typeof GroupRole
```

Comment:
```suggestion
	editable?: boolean
```
maybe change like this as the role isn't really needed here